app/src/main/res/values/strings.xml

```diff
@@ -197,7 +197,7 @@
     <string name="txt_spotify_api_authorization_success">Spotify Web API authorized!</string>
     <string name="txt_spotify_api_authorization_canceled">Spotify Web API authorization canceled.</string>
     <string name="txt_spotify_api_authorization_failed">Spotify Web API authorization failed. Please try again.</string>
-    <string name="lbl_spotify_api_authorized">Spotify Web API Authorized</string>
+    <string name="lbl_spotify_api_authorized">Ability to search Spotify</string>
```

Comment:
Ideally this should be a different string key, so that the (currently only 3) translations know to translate a fresh new string.

Reply:
Oh yeah you're right, reverted this change and added a new string with the "Ability to search Spotify" message that is being used